### ISSUE TITLE
TAB for mod doesn't work with overlays and accessibility and stuff

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#4911]
Eating 'tab' to toggle the mod as I have it coded

1: means tab when the store overlay up doesn't swap fields and
2: gets in the way of accessibility keyboard traversal somewhat

i need to ponder this especially the second so wanted to write it down before i forgot.
